jenkins.sh

```diff
@@ -13,12 +13,20 @@ rm -f gemfiles/*.gemfile.lock
 # Exclude /tmp from git clean as it only contains the signonotron checkout
 git clean -fdxe /tmp
 
-RBENV_VERSION=1.9.3 bundle install --path "${HOME}/bundles/${JOB_NAME}"
-RBENV_VERSION=2.1 bundle install --path "${HOME}/bundles/${JOB_NAME}"
+for ruby_version in 1.9.3 2.1; do
+  for gemfile in rails_3.2 rails_4.0 rails_4.1; do
+    RBENV_VERSION=${ruby_version} bundle install \
+      --path "${HOME}/bundles/${JOB_NAME}" \
+      --gemfile "gemfiles/${gemfile}.gemfile"
 
-RBENV_VERSION=1.9.3 bundle exec rake
-RBENV_VERSION=2.1 bundle exec rake
+    RBENV_VERSION=${ruby_version} \
+      BUNDLE_PATH="${HOME}/bundles/${JOB_NAME}" \
+      BUNDLE_GEMFILE="gemfiles/${gemfile}.gemfile" \
+      bundle exec rake
+  done
+done
 
 if [[ -n "$PUBLISH_GEM" ]]; then
+  bundle install
```

Comment:
It'll pick up the current contents of `.bundle/config`, meaning the last `bundle install`'s path.